### PR TITLE
Added missing Turkish translations, fixed wrong lines.

### PR DESCRIPTION
**Pull request by miratcan**

### CODE REVIEW

**Comment by grotewold** [comment]
Thanks!

**Comment by miratcan** [comment]
When can I see it online?

**Comment by Continuities** [comment]
I'll deploy it today! (Unless I forget...)

**Comment by miratcan** [comment]
Did you? :)

**Comment by Continuities** [comment]
I did... I'll do it today!

**Comment by Continuities** [comment]
It's deployed!

File: lang/tr/strings.po [hunk 1]
```diff
@@ -996,47 +990,38 @@ msgid "not enough leather"
...
 #: ../../script/localization.js:57
-#, fuzzy
 msgid "not enough meat"
-msgstr "deri yetmiyor"
+msgstr "yetersiz et"
 
 #: ../../script/localization.js:58
-#, fuzzy
 msgid "the compass points east"
 msgstr "pusula doğuyu gösteriyor."
```
Comment (kutaygs) [inline lang/tr/strings.po]: ("pusula kuzeyi gösteriyor") its the right one!!

**Pull request merged**